test/insert_typed_integration_test.go

```diff
@@ -0,0 +1,135 @@
+package test
+
+import (
+	"encoding/json"
+	"log"
+	"sync"
+	"testing"
+
+	"github.com/Ringloop/mr-plow/elastic"
+	"github.com/Ringloop/mr-plow/movedata"
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/assert"
+)
+
+type insertTypedIntegrationTest struct{}
+
+// test case config scenario
+func (*insertTypedIntegrationTest) ReadConfig() ([]byte, error) {
+
+	testComplexConfig := `
```

Comment:
maybe rename this

Reply:
which should I rename?

Reply:
minor: `testComplexConfig` is not in scope with the type `insertTypedIntegrationTest` ?

Reply:
Done